src/components/expandable-help/ExpandableHelp.vue

```diff
@@ -11,7 +11,7 @@
       <span class="pl-2 help-label">{{ !helpToggle ? helpLabel : 'Hide Help' }}</span>
     </div>
 
-    <v-expand-transition>
+    <v-slide-y-transition hide-on-leave>
```

Comment:
I also tried "expand" with `hide-on-leave` (which has no effect), and "scale" (which explodes on the screen kinda weird), and "scroll-y" (which looks almost the same as slide-y. Ultimately, "slide-y" with the option looks best to me.